frontend/src/components/Candidate-Decider/ApplicantCredentials.tsx

```diff
@@ -12,6 +12,55 @@ type Props = {
   preferredName?: string;
 };
 
+const fixLink = (link: string, git: boolean, linkedIn: boolean): string | undefined => {
+  if (!link) {
+    return undefined;
+  }
+
+  const urlRegex =
+    /(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_+.~#?&//=]*)/g;
+  const matches = link.match(urlRegex);
+
+  let extractedLink = matches ? matches[0].trim() : link.trim();
+
+  let fixedLink: string = '';
+
+  if (!extractedLink.startsWith('https://') && extractedLink.startsWith('http://')) {
+    extractedLink = extractedLink.replace('http://', 'https://');
+  }
+
+  if (extractedLink.includes('https://')) {
```

Comment:
I would expect that at this point, I know I'm working with some type of link. The regex + trim should've extracted out the link already, so I don't see a scenario where there would be "https://" in the string but not at the beginning. Let me know if I'm missing something though!